algod/build/install.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```

Comment:
This script is mostly identical to the one in sandbox. I needed to make a small change because the `SHA` option in sandbox is currently broken unless a branch is also provided.